data/settings/icon-grid-C.json.in

```diff
@@ -24,14 +24,9 @@
   "eos-folder-curiosity.directory": [
     "com.endlessnetwork.sciencesnacks.desktop",
     "com.endlessnetwork.shortfilms.desktop",
-    "com.endlessm.celebrities.en.desktop",
```

Comment:
Sorry, I should have been clearer in my comment. These icon grids are still used in the OS, so we should keep these app IDs in these files - we still want these apps to appear in these folders on the default desktop.